#########################################################################/**
# @RdocData hg19.1000kbp.SR50
#
# @title "hg19 1000kbp bin annotations"
# 
# \description{ The annotated data frame contains pre calculated data for
# each bin describing:
#   * chromosome: Chromosome name,
#   * start: Base pair start position,
#   * end: Base pair end position,
#   * bases: Percentage of non-N nucleotides (of full bin size),
#   * gc: Percentage of C and G nucleotides (of non-N nucleotides),
#   * mappability: Average mappability of 50mers with a maximum of 2 mismatches,
#   * blacklist: Percent overlap with ENCODE blacklisted regions,
#   * residual: Median loess residual from 1000 Genomes (50mers),
#   * use: Whether the bin should be used in subsequent analysis steps),
# }
#
# @author "DS"
#*/#########################################################################

#########################################################################/**
# @RdocData hg19.500kbp.SR50
#
# @title "hg19 500kbp bin annotations"
# 
# \description{ The annotated data frame contains pre calculated data for
# each bin describing:
#   * chromosome: Chromosome name,
#   * start: Base pair start position,
#   * end: Base pair end position,
#   * bases: Percentage of non-N nucleotides (of full bin size),
#   * gc: Percentage of C and G nucleotides (of non-N nucleotides),
#   * mappability: Average mappability of 50mers with a maximum of 2 mismatches,
#   * blacklist: Percent overlap with ENCODE blacklisted regions,
#   * residual: Median loess residual from 1000 Genomes (50mers),
#   * use: Whether the bin should be used in subsequent analysis steps),
# }
#
# @author "DS"
#*/#########################################################################

#########################################################################/**
# @RdocData hg19.100kbp.SR50
#
# @title "hg19 100kbp bin annotations"
# 
# \description{ The annotated data frame contains pre calculated data for
# each bin describing:
#   * chromosome: Chromosome name,
#   * start: Base pair start position,
#   * end: Base pair end position,
#   * bases: Percentage of non-N nucleotides (of full bin size),
#   * gc: Percentage of C and G nucleotides (of non-N nucleotides),
#   * mappability: Average mappability of 50mers with a maximum of 2 mismatches,
#   * blacklist: Percent overlap with ENCODE blacklisted regions,
#   * residual: Median loess residual from 1000 Genomes (50mers),
#   * use: Whether the bin should be used in subsequent analysis steps),
# }
#
# @author "DS"
#*/#########################################################################

#########################################################################/**
# @RdocData hg19.50kbp.SR50
#
# @title "hg19 50kbp bin annotations"
# 
# \description{ The annotated data frame contains pre calculated data for
# each bin describing:
#   * chromosome: Chromosome name,
#   * start: Base pair start position,
#   * end: Base pair end position,
#   * bases: Percentage of non-N nucleotides (of full bin size),
#   * gc: Percentage of C and G nucleotides (of non-N nucleotides),
#   * mappability: Average mappability of 50mers with a maximum of 2 mismatches,
#   * blacklist: Percent overlap with ENCODE blacklisted regions,
#   * residual: Median loess residual from 1000 Genomes (50mers),
#   * use: Whether the bin should be used in subsequent analysis steps),
# }
#
# @author "DS"
#*/#########################################################################

#########################################################################/**
# @RdocData hg19.30kbp.SR50
#
# @title "hg19 30kbp bin annotations"
# 
# \description{ The annotated data frame contains pre calculated data for
# each bin describing:
#   * chromosome: Chromosome name,
#   * start: Base pair start position,
#   * end: Base pair end position,
#   * bases: Percentage of non-N nucleotides (of full bin size),
#   * gc: Percentage of C and G nucleotides (of non-N nucleotides),
#   * mappability: Average mappability of 50mers with a maximum of 2 mismatches,
#   * blacklist: Percent overlap with ENCODE blacklisted regions,
#   * residual: Median loess residual from 1000 Genomes (50mers),
#   * use: Whether the bin should be used in subsequent analysis steps),
# }
#
# @author "DS"
#*/#########################################################################

#########################################################################/**
# @RdocData hg19.15kbp.SR50
#
# @title "hg19 15kbp bin annotations"
# 
# \description{ The annotated data frame contains pre calculated data for
# each bin describing:
#   * chromosome: Chromosome name,
#   * start: Base pair start position,
#   * end: Base pair end position,
#   * bases: Percentage of non-N nucleotides (of full bin size),
#   * gc: Percentage of C and G nucleotides (of non-N nucleotides),
#   * mappability: Average mappability of 50mers with a maximum of 2 mismatches,
#   * blacklist: Percent overlap with ENCODE blacklisted regions,
#   * residual: Median loess residual from 1000 Genomes (50mers),
#   * use: Whether the bin should be used in subsequent analysis steps),
# }
#
# @author "DS"
#*/#########################################################################

#########################################################################/**
# @RdocData hg19.10kbp.SR50
#
# @title "hg19 10kbp bin annotations"
# 
# \description{ The annotated data frame contains pre calculated data for
# each bin describing:
#   * chromosome: Chromosome name,
#   * start: Base pair start position,
#   * end: Base pair end position,
#   * bases: Percentage of non-N nucleotides (of full bin size),
#   * gc: Percentage of C and G nucleotides (of non-N nucleotides),
#   * mappability: Average mappability of 50mers with a maximum of 2 mismatches,
#   * blacklist: Percent overlap with ENCODE blacklisted regions,
#   * residual: Median loess residual from 1000 Genomes (50mers),
#   * use: Whether the bin should be used in subsequent analysis steps),
# }
#
# @author "DS"
#*/#########################################################################

#########################################################################/**
# @RdocData hg19.5kbp.SR50
#
# @title "hg19 5kbp bin annotations"
# 
# \description{ The annotated data frame contains pre calculated data for
# each bin describing:
#   * chromosome: Chromosome name,
#   * start: Base pair start position,
#   * end: Base pair end position,
#   * bases: Percentage of non-N nucleotides (of full bin size),
#   * gc: Percentage of C and G nucleotides (of non-N nucleotides),
#   * mappability: Average mappability of 50mers with a maximum of 2 mismatches,
#   * blacklist: Percent overlap with ENCODE blacklisted regions,
#   * residual: Median loess residual from 1000 Genomes (50mers),
#   * use: Whether the bin should be used in subsequent analysis steps),
# }
#
# @author "DS"
#*/#########################################################################

#########################################################################/**
# @RdocData hg19.1kbp.SR50
#
# @title "hg19 1kbp bin annotations"
# 
# \description{ The annotated data frame contains pre calculated data for
# each bin describing:
#   * chromosome: Chromosome name,
#   * start: Base pair start position,
#   * end: Base pair end position,
#   * bases: Percentage of non-N nucleotides (of full bin size),
#   * gc: Percentage of C and G nucleotides (of non-N nucleotides),
#   * mappability: Average mappability of 50mers with a maximum of 2 mismatches,
#   * blacklist: Percent overlap with ENCODE blacklisted regions,
#   * residual: Median loess residual from 1000 Genomes (50mers),
#   * use: Whether the bin should be used in subsequent analysis steps),
# }
#
# @author "DS"
#*/#########################################################################

# EOF